HACKING.rst

```diff
@@ -0,0 +1,61 @@
+Developing Sensu Go Ansible Collection
+======================================
+
+So, you have decided to help us out. Great! Let us set up a development
+environment together, and then you can start hacking ;)
+
+Preparing a development environment
+-----------------------------------
+
+The first thing we need to do is create a new virtual environment and activate
+it::
+
+   $ cd /path/to/desired/directory
+   $ python3 -m venv venv
+   (venv) $ . venv/bin/activate
+
+Next, we need to clone the source code::
+
+   (venv) $ mkdir -p ansible_collections/sensu
+   (venv) $ cd ansible_collections/sensu
+   (venv) $ git clone git@github.com:sensu/sensu-go-ansible.git sensu_go
+
+It is vitally important that we create two parent directories before checking
+out the code and that you clone the code into the `sensu_go` directory.
+Ansible development tools often assume that we are working from the
+``ansible_collections/<namespace>/<collection>`` directory.
+
+Now we need to install Ansible. At the moment, we can only use prerelease
+Ansible versions greater or equal to ``2.9.0rc4``. We can get the appropriate
+version by running::
+
+   (venv) $ pip install --pre ansible
+
+All that separates now from the fully functioning development environment is a
+few dependencies. We can install them by running the next command::
+
+   (venv) $ pip install \
+              -r sanity.requirements \
+              -r units.requirements \
+              -r integration.requirements
+
+And this is it. We are all set now. To validate our setup, we can run the
+bundled tests::
+
+   (venv) $ ansible-test sanity --python 3.7
```

Comment:
Maybe we could add a note for the user to inspect the output of `python -V` (and check `ansible-test sanity --help` to see which versions are allowed for the `--python` argument) so that the user picks the correct value, which must match the version of interpreter installed in his environment. 

Feel free to ignore this if you think it's a bit of an overkill :slightly_smiling_face: